server/src/main/java/com/tencent/rss/server/ShuffleServerMetrics.java

```diff
@@ -59,6 +65,14 @@ public class ShuffleServerMetrics {
   private static final String TOTAL_HDFS_WRITE_DATA = "total_hdfs_write_data";
   private static final String TOTAL_LOCALFILE_WRITE_DATA = "total_localfile_write_data";
   private static final String TOTAL_REQUIRE_BUFFER_FAILED = "total_require_buffer_failed";
+  private static final String LOCAL_STORAGE_TOTAL_WRITE = "local_storage_total_write";
```

Comment:
It is possible to use the unified prefix but the different suffix (local or remote host)?

Reply:
ok